avocado/core/runners/avocado_instrumented.py

```diff
@@ -72,24 +24,6 @@ class and method names should be separated by a ":".  One
     """
     DEFAULT_TIMEOUT = 86400
 
-    @staticmethod
```

Comment:
While I understand that this is also related to "messages", IMO it should for now remain here.  Reasons include:

1. there's no usage elsewhere
2. its implementation is really geared towards Avocado's instrumented tests behavior

Reply:
I moved this to utils, because the same method is also used in vt runner. I moved this to utils, because the same method is also used in vt runner. I suppose that this can be useful for other runners, when those runners are based on avocado instrumented tests like avocado-vt.

Reply:
OK, I did not think of the VT runner. So, that makes my first point incorrect.

Reply:
And actually, the second point too. :)